src/Simplex/Chat/Store.hs

```diff
@@ -565,9 +551,15 @@ deleteContactConnectionsAndFiles db userId Contact {contactId} = do
 deleteContact :: DB.Connection -> UserId -> Contact -> IO ()
 deleteContact db userId Contact {contactId, localDisplayName} = do
   DB.execute db "DELETE FROM chat_items WHERE user_id = ? AND contact_id = ?" (userId, contactId)
-  deleteContactProfile_ db userId contactId
+  ctMembers :: [ContactId] <- map fromOnly <$> DB.query db "SELECT contact_id FROM group_members WHERE user_id = ? AND contact_id = ?" (userId, contactId)
```

Comment:
why do you need the list, not count?

Reply:
also, there probably should be some similar extra logic when deleting members - if the profile is no longer used, remove it too.

Reply:
and local display name

Reply:
and when deleting groups